### PR TITLE
Increase width for country code dropdown

### PR DESCRIPTION
**Pull request by swapnilraj**
#### 3 digit country code
image
image

#### 2 digit country code
image
image

#### 1 digit country code
image
image

Fixes the width of the drop-down menu for country code.
The initial width seems like an arbitrary number, so I just increased it to fit.

Signed-off-by: Swapnil Raj sr@sraj.me

### CODE REVIEW

**Comment by JorikSchellekens** [comment]
This will probably break with font scaling. Can you change it to `$font-78px` and the correct mapping to `_font-sizes.scss`

**Comment by swapnilraj** [comment]
I was thinking about the same. Does this also imply that every other place that a component is using an absolute value, it should be using a corresponding value from `_font-sizes.scss`?

**Comment by JorikSchellekens** [comment]
No, we only want certain parts of the app to scale and scaling all of the padding and margins does not make for a good experience.

File: res/css/views/elements/_Field.scss [hunk 1]
```diff
@@ -16,6 +16,8 @@ limitations under the License.
 
 /* TODO: Consider unifying with general input styles in _light.scss */
 
+@import "../../_font-sizes.scss";
+
```
Comment (t3chguy) [inline res/css/views/elements/_Field.scss:20]: this is not needed, it is implicitly imported by the base style

Comment (JorikSchellekens) [reply res/css/views/elements/_Field.scss:20]: Nicely spotted

Comment (swapnilraj) [reply res/css/views/elements/_Field.scss]: done

**Pull request merged**